### PR TITLE
Update the owner and minor testing tweaks

### PR DESCRIPTION
**Pull request by tas50**
Add more modern platforms to dokken config
Yank travis config and badge
Add code of conduct
Update metadata ownership and remove mention of chef in the readme
Remove Ubuntu 14.04 configs in kitchen
Remove Chef 12 testing in Kitchen

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # nginx Cookbook
 
-[![Cookbook](http://img.shields.io/cookbook/v/nginx.svg)](https://supermarket.chef.io/cookbooks/nginx) [![Build Status](https://travis-ci.org/chef-cookbooks/nginx.svg?branch=master)](https://travis-ci.org/chef-cookbooks/nginx)
+[![Cookbook](http://img.shields.io/cookbook/v/nginx.svg)](https://supermarket.chef.io/cookbooks/nginx)
```
Comment (josephholsten) [inline README.md]: we'll want the circleci build badge soon

File: README.md [hunk 2]
```diff
@@ -366,13 +366,9 @@ node.run_state['nginx_configure_flags'] =
   node.run_state['nginx_configure_flags'] | ['--with-SOMETHING', "--with-SOME_OPT='things'"]
 ```
 
-## Maintainers
```
Comment (josephholsten) [inline README.md]: would it be valuable for us to start having a boilerplate like this for sous? My first thought is "less code less problems", but this isn't really code...

Comment (tas50) [reply README.md]: It would be a good thing to add here since it shows up on supermarket

File: README.md (hunk 1 above)
Comment (tas50) [reply README.md]: We will once it's all setup

**Pull request merged**